Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.2
+MAINTAINER Mikhail Simin <mikhail@nextdoor.com>
+
+RUN mkdir -p /app /app/zk_monitor
+ADD zk_monitor/runserver.py /app/zk_monitor/runserver.py
```

Comment:
Umm .. this doesn't really do much though. You've added runserver.py, but *none of the other code*.